packages/calling-component-bindings/src/utils/callUtils.ts

```diff
@@ -103,6 +103,41 @@ export const _updateUserDisplayNames = (participants: RemoteParticipantState[]):
   }
 };
 
+/**
+ * Get the display name of a participant for given userId
+ *
+ * @internal
+ *
+ */
+export const getRemoteParticipantDisplayName = (
```

Comment:
nit: other places beyond display name in future may want remote participants ended.
A different way to structure this PR, would be to create a new selector `getAllRemoteParticipants` and then where `getRemoteParticipants, getRemoteParticipantsEnded` is used as argumants into the reselect, you can just use getAllRemoteParticipants knowing it will retrieve both active and previous participants.